src/Symfony/Component/Validator/Resources/translations/validators.ru.xlf

```diff
@@ -310,10 +310,26 @@
                 <source>This value does not match the expected {{ charset }} charset.</source>
                 <target>Значение не совпадает с ожидаемой {{ charset }} кодировкой.</target>
             </trans-unit>
+            <trans-unit id="81">
+                <source>This is not a valid Business Identifier Code (BIC).</source>
+                <target>Значение не соответствует формату BIC.</target>
```

Comment:
I think we can use Russian code name here, since it's always written as `БИК` in documents.
We should also expand it similarly to the English version. WDYT?

```suggestion
                <target>Значение не соответствует формату Банковского идентификационного кода (БИК).</target>
```

See [wikipedia article](https://ru.wikipedia.org/wiki/%D0%91%D0%B0%D0%BD%D0%BA%D0%BE%D0%B2%D1%81%D0%BA%D0%B8%D0%B9_%D0%B8%D0%B4%D0%B5%D0%BD%D1%82%D0%B8%D1%84%D0%B8%D0%BA%D0%B0%D1%86%D0%B8%D0%BE%D0%BD%D0%BD%D1%8B%D0%B9_%D0%BA%D0%BE%D0%B4).

Reply:
https://translate.academic.ru/%D0%91%D0%98%D0%9A/ru/en
Russian БИК is not the same as BIC, it's RCBIC

Reply:
BIC is more like 'Свифт код', but I'd prefer to leave it as it is now, looks definitely better